caffeine@patapon.info/extension.js

```diff
@@ -1106,9 +1106,16 @@ class Caffeine extends QuickSettings.SystemIndicator {
 });
 
 export default class CaffeineExtension extends Extension {
+    /**
+     * @param {ExtensionMeta} metadata - Caffeine extension metadata
+     */
+    constructor(metadata) {
+        super(metadata);
+    }
+
```

Comment:
```suggestion

```

Do we really need this? We aren't overriding `constructor()`, I don't think we need to do anything here.

Reply:
Absolutely not :-) sorry about that...